### ISSUE TITLE
get_oxidized_config API endpoint doesn't work when using groups

### ISSUE THREAD

**Issue by davromaniak** [librenms/librenms#14046]
### The problem

Hi.

When using the /api/v0/oxidized/config endpoint, with a valid device name (the same I get using /api/v0/oxidized), I receive this output : 

```
{
    "status": "ok",
    "config": "node not found"
}
```

The URL I call is : /api/v0/oxidized/config/valid-device-name

When I put anything invalid as the device name (even the actual device FQDN), I get this : 

```
{
    "status": "error",
    "message": "Received no data from Oxidized"
}
```

I use subfolders within the oxidized git repository output, don't know if it can cause any issue.

I don't see anything in /opt/librenms/logs

Thanks.

### Output of ./validate.php

```txt
====================================
Component | Version
--------- | -------
LibreNMS  | 22.5.0
DB Schema | 2022_04_08_085504_isis_adjacencies_table_add_index (236)
PHP       | 7.3.20
Python    | 3.6.8
MySQL     | 10.3.28-MariaDB
RRDTool   | 1.7.0
SNMP      | 5.8
====================================

[OK]    Composer Version: 2.3.5
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database schema correct
```

**Comment by davromaniak** [comment]
Hey.

If it can help, I did some research, and the issue comes from using groups.

Like it's written in Oxidized README : 
```
Restful API to fetch configurations (/node/fetch/[NODE] or /node/fetch/group/[NODE])
```

I tried locally with curl : 

Without the group : 
```
$ curl -s 'http://127.0.0.1:8888/node/fetch/valid-device-name?format=json'
"node not found"
```

With the group : 
```
$ curl -s 'http://127.0.0.1:8888/node/fetch/valid-group/valid-device-name?format=json' | wc -c
39937
```

When calling with the group, it's returning the good and full config.

I edited the issue title.

Thanks.

**Comment by electrocret** [comment]
This should be fixed by #14750
